src/main/java/org/sonar/plugins/crowd/CrowdGroupsProvider.java

```diff
@@ -94,7 +96,12 @@ public CrowdGroupsProvider(CrowdClient crowdClient) {
     return ImmutableList.copyOf(groups);
   }
 
-  @Override
```

Comment:
Shouldn't this `@Override` be on the groups provider on the sonar 9.0 new method?

Reply:
Will switch to 8.9 next week and check the overrides.

Reply:
I've just tested with 8.9 in the pom and with the overrides, it works on SQ 9.0, I'll test on actual 8.9 and check that the groups are synced tomorrow or when the icecream cools me down